groups/ntc/ntci/ntci_listenersocketmanager.h

```diff
@@ -19,13 +19,8 @@
 #include <bsls_ident.h>
 BSLS_IDENT("$Id: $")
 
-#include <ntccfg_platform.h>
```

Comment:
What is the rationale for removing these headers? Did you discover they are included transitively? For sanity, I made a policy to have every header include ntccfg_platform.h and ntcscm_version.h (the former being an NTF policy and the latter being a BDE policy, actually.)

Reply:
I returned back all the removed includes. I just don't see a point in having them if they are not used in that particular file.
The way I discovered it - when I was modifying the file my IDE marked them as "unused includes".